src/components/molecules/card/AnyWayCard.tsx

```diff
@@ -81,6 +88,13 @@ const AnyWayCard: FC<IProps> = ({
       <AnyWayButton className={classes.button} disableRipple={true} onClick={handleCardEditorOpen}>
         <SettingsOverscanIcon />
       </AnyWayButton>
+      <Tooltip title={LOREM_IPSUM} placement="top" aria-label="info">
+        <span>
+          <AnyWayButton className={`${classes.button} ${classes.hidden}`} disableRipple={true} onClick={() => {}}>
```

Comment:
The button is not required here. you can use the `span` for the classes (or even better - use `box`)